tests/api/v1/generatorTemplates/get.js

```diff
@@ -0,0 +1,372 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/api/v1/generators/get.js
+ */
+'use strict';
+
+const supertest = require('supertest');
+const api = supertest(require('../../../../index').app);
+const constants = require('../../../../api/v1/constants');
+const tu = require('../../../testUtils');
+const u = require('./utils');
+const GeneratorTemplate = tu.db.GeneratorTemplate;
+const path = '/v1/generatorTemplates';
+const expect = require('chai').expect;
+const ZERO = 0;
+const ONE = 1;
+const TWO = 2;
+const THREE = 3;
+
+describe(`api: GET ${path}`, () => {
```

Comment:
This one just came out of the review in #525 .------ "the outermost "describe" for every test should be named for the file it's in (and every "describe" name should end with " >")" . We can choose to correct it now or later.